packages/gatsby-theme-api-docs/src/components/api/resource/method/method.js

```diff
@@ -65,89 +65,84 @@ const Method = ({ apiKey, url, resourceUriParameters, method, methodType }) => {
     allUriParameters = allUriParameters.concat(method.uriParameters);
   }
 
+  const methodColor = computeMethodColor(methodType.toLowerCase());
+
+  const id = generateEndpointURN({
+    apiKey,
+    path: new URL(url).pathname,
+    method: methodType,
+  });
+
   return (
-    <div
-      id={generateEndpointURN({
-        apiKey,
-        path: new URL(url).pathname,
-        method: methodType,
-      })}
-    >
-      <SpacingsStack scale="s">
-        <SpacingsInline alignItems="center" scale="m">
-          <Type
-            css={computeMethodNameBackgroundColor(methodType.toLowerCase())}
-          >
-            {methodType}
-          </Type>
-
-          <Title>{method.displayName}</Title>
-        </SpacingsInline>
-
-        <Description>{method.description}</Description>
-
-        <SpacingsStack scale="m">
-          <UrlScopesResponseOverallContainer>
-            <SpacingsInline alignItems="stretch">
-              <UrlScopesResponseContainerStart
-                css={computeMethodNameBackgroundColor(methodType.toLowerCase())}
-              />
-              <UrlScopesResponses
-                data={{
-                  url,
-                  scopes: {
-                    title: oauth2Scopes,
-                    scopes: method.securedBy
-                      ? method.securedBy[0].oauth_2_0.scopes
-                      : null,
-                  },
-                  responses: method.responses,
-                }}
-              />
-            </SpacingsInline>
-          </UrlScopesResponseOverallContainer>
-
-          {allUriParameters.length > 0 ? (
-            <Parameters
-              title={pathParametersTitle}
-              parameters={allUriParameters}
-            />
-          ) : null}
-
-          {method.queryParameters ? (
-            <Parameters
-              title={queryParametersTitle}
-              parameters={method.queryParameters}
-            />
-          ) : null}
-
-          {method.body ? (
-            <RequestRepresentation
-              titleSuffix={requestRepresentation}
-              apiKey={apiKey}
-              apiType={method.body.applicationjson.type}
-            />
-          ) : null}
-        </SpacingsStack>
+    <SpacingsStack scale="s">
+      <SpacingsInline alignItems="center" scale="s">
+        <Type
+          css={css`
+            background-color: ${methodColor};
+          `}
+        >
+          {methodType}
+        </Type>
+
+        <TitleWithAnchor id={id}>{method.displayName}</TitleWithAnchor>
+      </SpacingsInline>
+
+      <Description>{method.description}</Description>
+
+      <SpacingsStack scale="m">
+        <UrlScopesResponseContainer
+          css={css`
+            border-left-color: ${methodColor};
+          `}
+        >
+          <UrlScopesResponses
+            data={{
```

Comment:
Any particular reason for passing the data as a single prop instead of multiple props for each data property?

Reply:
There was a reason, I don't really remember exactly why at the moment.

Reply:
This question is still valid. I would spread the `data` into single props, and be explicit which props are passed to the component

```js
<UrlScopesResponses
  apiKey={apiKey}
  uris={uris}
  scopes={{ }}
  responses={method.responses}
/>
```

Reply:
I guess the question from me is, what is the problem?

Reply:
In general, it's always better to pass scalar values as props, which makes the component easier to optimize for "performance".

Why I also asked was that you are constructing the object to be passed as a single prop from values that are already separated, while then destructuring it again in the component itself. It does not make much sense to me.

Reply:
I will change it but your reason doesn't really make sense to me.

Reply:
You don't need to change it because I said so, I wanted to understand what was the reason for doing that as it looked "weird" to me doing it that way.
I then explain how and why I would do it.